reana_client/api/client.py

```diff
@@ -670,8 +670,10 @@ def get_workflow_specification(workflow, access_token):
         raise e
 
 
-def delete_workflow(workflow, all_runs, workspace, access_token):
-    """Delete a workflow."""
+def delete_workflow(workflow, all_runs: bool, workspace: bool, access_token: str):
+    """Delete a workflow. Forces to delete workspace."""
+    workspace = True
```

Comment:
I think it might be a good idea to print a warning when `workspace` is set to `False`, what do you think?

Reply:
We do not do it with CLI so I am not sure.

Reply:
I think this is a little bit different from the CLI case. In the CLI case, the user cannot specify that he would like to keep the workspace (i.e. there is no `--keep-workspace` or a similar flag). Instead, in this case the user can call the function `delete_workflow(..., ..., False, ...)`: even though he has specified he would like to keep the workspace intact, we delete it anyway.

In this case, I think it makes sense to remove the `workspace` parameter.

What do you think about this? cc @tiborsimko

Reply:
I added a warning to the `delete_workflow`. Ideally would be good to remove the `workspace` parameter if it is no longer required.

Reply:
I would say let's just go with the warning message. PR has been stuck for a long time now.